### DISCUSSION TITLE
Recording streaming digital audio from a website

### DISCUSSION THREAD

**Discussion by muesic** [ExistentialAudio/BlackHole#644] [Q&A] [answered]
I was wondering if BlackHole could take e.g. the digital audio being streamed from a website (e.g. a podcast) and loop it back fully in the digital domain to be subsequently saved as a file? This would essentially just be creating an equivalent duplicate of the digital audio file that was "playing" from the website. I appreciate that the podcast's digital audio may still be converted to analog and played through the speakers etc., but as far as the recording of it goes, no DAC and ADC were involved in turning the original digital audio into analog and then back into digital prior to saving as an audio file. Is this possible? Part of the allure of doing this is that the audio levels etc. of the original source are all maintained whereas if it were done in the analog domain there is ample opportunity to screw things up by adding gain/attenuation into the various analog paths.

**Comment by devinroth** [comment, marked as answer]
You’re way over thinking this. Just download the audio file directly. There are plenty of websites that can do this for you. No conversation necessary. That being said, some people want to record into a daw which is where BlackHole will help you. Set output audio to BlackHole be then record from BlackHole in something like Audacity.

Devin

**Comment by yujinqiu** [comment]
@muesic I have the same need and I figure out  [BeMyEars](https://www.bemyears.io/) is perfect, it can recording the voice from BlackHole and export it as m4a file.
